subr/check_version.sh

```diff
@@ -0,0 +1,8 @@
+export selfupdate_macports="false"
+export macports_version="$1"
```

Comment:
`export` adds variables to a list that is transmitted to subprocesses, it's useful to pass parameters to subprocesses but I do not think this is your intent here.